test/QueryPlannerTestHelpers.h

```diff
@@ -93,19 +93,32 @@ inline auto TextIndexScanForWord = [](Variable textRecordVar,
   return RootOperation<::TextIndexScanForWord>(AllOf(
       AD_PROPERTY(::TextIndexScanForWord, getResultWidth,
                   Eq(1 + word.ends_with('*'))),
-      AD_PROPERTY(::TextIndexScanForWord, textRecordVar, Eq(textRecordVar)),
-      AD_PROPERTY(::TextIndexScanForWord, word, word)));
+      AD_PROPERTY(::TextIndexScanForWord, getTextRecordVar, Eq(textRecordVar)),
+      AD_PROPERTY(::TextIndexScanForWord, getWord, word)));
```

Comment:
I think we typically don't need the `get` if it doesn't compute anything expensively but just returns a (possibly const) reference.
(Check in both scans for consistency).